authors/apps/base/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.home, name='home'),
+]
```

Comment:
Add a new blank line at the end of file, that's as per pep8 style guide @charitymarani

Reply:
@Nicanor008 working on it